controllers/cartController.js

```diff
@@ -0,0 +1,83 @@
+const cartService = require("../services/cartService");
+
+const cartList = async (req, res) => {
+  try {
+    const list = await cartService.cartList(req.userId);
+
+    return res.status(201).json({ data: list });
+  } catch (err) {
+    console.log(err);
+    return res.status(err.statusCode || 500).json({ message: err.message });
+  }
+};
+
+const addCartInDetailPage = async (req, res) => {
```

Comment:
페이지에 의존하는 API 개발이 아니기 때문에 페이지 이름을 넣어줄 필요가 없습니다.
```suggestion
const addCart = async (req, res) => {
```